invenio_mail/ext.py

```diff
@@ -31,7 +31,22 @@
 
 from flask_mail import Mail, email_dispatched
 
-PY3 = sys.version_info[0] == 3
+
+def print_email(message, app):
+    """
+    Print mail to stream.
```

Comment:
Please start docstring on the same line: `"""Print mail to stream.\n\nSignal handler ...`